aquadoggo/src/graphql/client/static_types/next_entry_arguments_type.rs

```diff
@@ -10,7 +10,7 @@ use crate::graphql::scalars;
 
 /// Arguments required to sign and encode the next entry for an author.
 #[derive(SimpleObject, Serialize, Deserialize, Debug, Eq, PartialEq)]
-pub struct NextEntryArguments {
+pub struct NextEntryArgumentsType {
```

Comment:
You mentioned that on the chat, I think it's cool to rename all scalars to contain a "Scalar" suffix ("Type" is too abstract).

I'm also fine with _not_ doing that. So far I circumvented confusion by only importimg scalars together with the parent module ala `scalars::NextEntryArguments` which worked for me, but I understand that this is very subjective 😅

Reply:
I removed all the suffixes from types that are not representing some other type with the same name (e.g. `NextEntryArguments`).

For the scalars that wrap p2panda types with the exact same name I think it will be helpful to have them. I saw your approach with `scalars::X` and that also works! It's just another thing you have to keep in mind when writing code so maybe we can help panda devs in the future by adding `Scalar` or whatever GQL type is being represented as a suffix.

Reply:
Hah, I have finally found the issue. The scalar types all use a default implementation of async graphql's `Scalar` trait, which doesn't validate. So I will add that instead.